R/AutoGeneratedDefinitions.R

```diff
@@ -241,19 +241,23 @@ isValidCohortId <- function(cohortIds, baseUrl) {
 #' @details
 #' Obtain the Cohort definition from WebAPI for a given Cohort id
 #'
-#' @template BaseUrl
+#' @param wc         A WebApiConnection object 
```

Comment:
Hi @ablack3 this file is designed to be autogenerated

https://github.com/OHDSI/ROhdsiWebApi/blob/3e09d7350984c4ec9b4a93678b3492b5fcbf67ef/R/AutoGeneratedDefinitions.R#L18

From here
https://github.com/OHDSI/ROhdsiWebApi/blob/3e09d7350984c4ec9b4a93678b3492b5fcbf67ef/extras/PackageMaintenance.R#L57

So - when you make changes please consider a solution that uses the template based approach